paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/settings/LinkControllerAllowUserEmailEditsSettingsDefinition.kt

```diff
@@ -0,0 +1,23 @@
+package com.stripe.android.paymentsheet.example.playground.settings
+
+import com.stripe.android.link.LinkController
+import com.stripe.android.paymentsheet.example.playground.PlaygroundState
+
+internal object LinkControllerAllowUserEmailEditsSettingsDefinition : BooleanSettingsDefinition(
+    key = "linkControllerAllowUserEmailEdits",
+    displayName = "LinkController: allow user to edit email",
+    defaultValue = true,
+) {
```

Comment:
[follow-up] if viable, let's maybe create a separate playground activity for `LinkController`

Reply:
Way ahead of ya 😉 https://github.com/stripe/stripe-android/pull/11144